framework/tlvf/src/include/tlvf/ClassList.h

```diff
@@ -11,10 +11,10 @@
 #define _ClassList_H_
 
 #include <memory>
-#include <tlvf/BaseClass.h>
```

Comment:
There's no need to remove this include.

Reply:
But is there any need to keep it if it's not used? I think not.